### ISSUE TITLE
Issue with the --headless argument not working

### ISSUE THREAD

**Issue by archfz** [wix/Detox#2046]
**Describe the bug**
The --headless argument is not working since the arguments parser is verifying for === "true" while the argument will actually be boolean true. I have added in the local-cli/test.js `(program.headless ? '--headless "true"' : ''),` and this fixes the issue. But most likely this issue affects other parts of the application as well.

I am trying to run it in docker for CI and this is what I get:
```text
Error: Command failed: node_modules/.bin/mocha --opts test/integration/mocha/mocha.opts --configuration android.emu.release --cleanup --grep :ios: --invert --headless --use-custom-logger "true" "test/integration/specs"

Fatal: This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

**To Reproduce**

- [x ] I have tested this issue on the latest Detox release and it still reproduces

Steps to reproduce:
1. Install latest detox with mocha runner.
2. Attempt to run detox test --headless.
3. Notice in EmualtorExec.js that it is not "true" but rather true.

**Expected behavior**
The --headless command should properly work and set the -no-window argument for calling the emulator.

**Workaround**

Add in init.js of mocha before the require of detox this to temporarly apply fix until this is fixed in detox:
```js
// @TODO: Remove once https://github.com/wix/Detox/issues/2046 is fixed.
if (process.argv.includes('--headless')) {
  process.argv[process.argv.indexOf('--headless')] = '--headless=true';
  console.log(process.argv);
}

/// ...requires
```

**Environment (please complete the following information):**
 - Detox: 16.4, tested 16.2 as well
 - React Native: 0.62.2: does not matter
 - Node: 12.16.3
 - Device: desktop / docker: does not matter
 - OS: Ubunut 18.04: does not matter
 - Test-runner (select one): Mocha

**Comment by j320** [comment]
I'm also experiencing this. Using the --headless command launches the Android emulator normally with the UI present.

**Comment by noomorph** [comment]
Thanks for the report. Looking into it.

**Comment by noomorph** [comment]
Will be fixed in 16.7.1
